include/tvm/Variable.h

```diff
@@ -62,14 +67,19 @@ class TVM_DLLAPI Variable : public tvm::internal::ObjWithId, public std::enable_
    * derived from an other).
    */
   const Space & space() const;
+  /** Return the space of the subvariable preceeding this variable w.r.t its
+   * supervariable. E.g if this variable is x, and is a subvariable of X such
+   * that X = (u, x, y), x->spaceShift() corresponds to u->space().
+   */
+  const Space & spaceShift() const;
```

Comment:
This is an access to the third argument provided when calling `subvariable`, i.e. an important element of definition of the object. As such, it seems useful to expose it, especially for such a core class that will be widely used.

I might not use it for now because I directly use `shift_` or rely on higher level functions using it, but I see at least one clear use case: when you have a subvariable `x` in `X` and you want to get/create the subvariable preceding `x` in `X` (`u` in the documentation).